_includes/list_of_educators.html

```diff
@@ -14,11 +14,14 @@ <h3 id="{{ loopyear }}"> {{ loopyear }} </h3>
 We need to keep track of how many educators are skipped in order to have the
 proper index for the table column. -->
 {% assign skipped = 0 %}
+{% assign educators_added = false %}
 {% for person in site.profiles| sort:"title" %}
 {% unless person.training_years contains loopyear %}
 {% assign skipped = skipped | plus: -1 %}
 {% continue %}
-{% endunless %}
+{% else %}
+{% assign educators_added = true %}
+{% endif %}
```

Comment:
shouldn't that be an `endunless`?

Reply:
Added

Reply:
Sorry, I meant that the `endif` should be an `endunless`, not to add another `endunless` before

Reply:
done